.pre-commit-config.yaml

```diff
@@ -16,7 +16,6 @@
   - id: fix-encoding-pragma
   - id: flake8
     args: ["--max-line-length=88"]
-  - id: name-tests-test
```

Comment:
This doesn't look related + you can also use `args: [--django]` to invert the check

Reply:
Sorry, it is really not full related, poetry by default create a directory for tests and I simply remove this line from config.
It
https://github.com/serhii73/ukrdict/tree/feature/add_poetry/tests
and it
```
id: name-tests-test
```
is incompatible.

Reply:
Nope. It's compatible.